src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -725,6 +725,8 @@ def make_cell_suspensions(self, entry):
     def make_cell_line(self, cell_line):
         return {
             "id": cell_line["biomaterial_id"],
+            "cellLineType": cell_line.get("cell_line_type", None),
```

Comment:
I'm surprised that these two additional keys don't require some change in a service test assertion.

Reply:
Thought this was covered by existing test cases however turns out they are checking the response from projects with no cell_lines. Added a new test case to fix this missing coverage.